### DISCUSSION TITLE
LibreSBC Install Issue

### DISCUSSION THREAD

**Discussion by inoutglobal** [hnimminh/libresbc#173] [General]
Hello!

I tried to install LibreSBC by using Ansible, without success.

Following attached the install log output.
[libresbc_ansible_install_on_Debian10.log](https://github.com/user-attachments/files/16969798/libresbc_ansible_install_on_Debian10.log)

**Comment by hnimminh** [comment]
@inoutglobal It seems SignalWise change something on dependency for `buster`, can you retry with `bullsyes` (buster is EOL already)

**Reply by inoutglobal** [reply]
Dear, thanks! 

Sorry, I am not following you. Do i have to change something related to Debian version inside Ansible scripts? 
By the Way: Should LibreSBC be installed on a host running Debian 10 or Debian 11?

Best Regards!

**Reply by hnimminh** [reply]
debian11 is support for longtime ago, you can found it here https://github.com/hnimminh/libresbc/wiki/Installation
